external/src/bitshuffle/iochain.cc

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
does the IO chain source have to be included?  It seems like these components are not needed, although I don't know how intertwined they are with the core source code.

Reply:
I think it's only required when OpenMP is enabled, but I think it'd require a few changes to remove it (the methods are no-ops without OpenMP).